src/utils/user_registry.ts

```diff
@@ -25,25 +26,30 @@ export const getUserId = (
 > =>
   tryCatch(
     () =>
-      apiClient.getUserIdByExternalId({
+      apiClient.getUserByExternalId({
         SubscriptionKey: subscriptionKey,
-        externalId
+        body: {
+          externalId
+        }
       }),
     toError
   )
     .mapLeft<IResponseErrorInternal | IResponseErrorForbiddenNotAuthorized>(
-      toResponseErrorInternal
+      err => {
+        logger.error(`USER REGISTRY getUserByExternalId: ${err.message}`);
+        return toResponseErrorInternal(err);
+      }
     )
     // Validation (Either) -> taskEither
-    .chain(_ =>
-      fromEither(_).mapLeft(errs =>
+    .chain(_ => {
+      return fromEither(_).mapLeft(errs =>
         toResponseErrorInternal(errorsToError(errs))
-      )
-    )
+      );
+    })
```

Comment:
we can remove `{}` and just return `fromEither` instead

Reply:
Fixed by https://github.com/pagopa/hub-spid-login-ms/pull/39/commits/00ec18a204e9b2fe7360f12a2594552226ae0445 @AleDore